lib/inspec/resources/virtualization.rb

```diff
@@ -236,6 +254,39 @@ def detect_lxd
       true
     end
 
+    # Detect VMware
+    def detect_vmware
+      return false unless inspec.file("/sys/devices/virtual/dmi/id/product_name").exist?
+
+      product_name = inspec.file("/sys/devices/virtual/dmi/id/product_name").content
+
+      if product_name =~ /^VMware/
+        @virtualization_data[:system] = "vmware"
+        @virtualization_data[:role] = "guest"
```

Comment:
I think it's fine to merge this as is, but it would be possible to save having the `else` branch by returning `return true` here then returning `false` if we reach the end of the method.

Reply:
Just been following the pattern of other helpers in that file :)